hudi-common/src/main/java/org/apache/hudi/common/table/HoodieTableMetaClient.java

```diff
@@ -324,7 +324,7 @@ public static HoodieTableMetaClient initTableType(Configuration hadoopConf, Stri
    */
   public static HoodieTableMetaClient initDatasetAndGetMetaClient(Configuration hadoopConf, String basePath,
       Properties props) throws IOException {
-    LOG.info("Initializing " + basePath + " as hoodie dataset " + basePath);
+    LOG.info("Initializing {} as hoodie dataset {}", basePath, basePath);
```

Comment:
change to `LOG.info("Initializing {} as hoodie dataset", basePath);`

Reply:
Thanks, change it now.